Xamarin.Essentials/Permissions/Permissions.android.cs

```diff
@@ -457,8 +457,22 @@ public partial class StorageRead : BasePlatformPermission
 
         public partial class StorageWrite : BasePlatformPermission
         {
-            public override (string androidPermission, bool isRuntime)[] RequiredPermissions =>
-                new (string, bool)[] { (Manifest.Permission.WriteExternalStorage, true) };
+            public override (string androidPermission, bool isRuntime)[] RequiredPermissions
+            {
+                get
+                {
+                    if (Platform.HasApiLevel(BuildVersionCodes.Q))
+                    {
+                        return new (string, bool)[]
+                        {
+                            ("android.permission.READ_MEDIA_IMAGES",  true),
```

Comment:
These will probably be needed to spit out into their own permissions.

My app only needs access to the users images, I don't want to add/request for 'Audio'/'Video' permissions